core/storage/diskstore_test.go

```diff
@@ -0,0 +1,98 @@
+package storage
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestDiskGetNoFileErrors(t *testing.T) {
```

Comment:
nit: I get wary when I see real I/O in unit tests, even with tempdirs. I think it's better to mock out the I/O. That being said, this isn't a huge issue. Mocking out is a bit of work.

Reply:
Hrm. I'm not sure if that's worth doing. If you want it done, I'm happy to do it.

Reply:
not a big deal